Tests/Resources/app/config/config.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Features\Context\ResourceContext;
+
+$container->setParameter('cmf_testing.bundle_fqn', 'Symfony\Cmf\Bundle\ResourceRestBundle');
+$loader->import(CMF_TEST_CONFIG_DIR.'/dist/parameters.yml');
+$loader->import(CMF_TEST_CONFIG_DIR.'/dist/framework.yml');
+$loader->import(CMF_TEST_CONFIG_DIR.'/dist/monolog.yml');
+$loader->import(CMF_TEST_CONFIG_DIR.'/dist/doctrine.yml');
+$loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
+
+if (file_exists(ResourceContext::getConfigurationFile())) {
+    $loader->import(ResourceContext::getConfigurationFile());
```

Comment:
My plan is to decouple it feom phpunit ans have its own configuration file. Thaat would make it way more usable, whether it's going to be a wrapper around testing frameworks or just a helper, I'm not sure about.